frontend/src/Routing/routes.js

```diff
@@ -3,3 +3,4 @@ export const AboutUsRoute = '/aboutus';
 export const ContactUsRoute = '/contactus';
 export const LoginRoute = '/login';
 export const SignUpRoute = '/signup';
+export const AlbumEndPointRoute = '/albumendpoint';
```

Comment:
rename both the variable and the endpoint

RecordRoute = '/record/'

Reply:
you could also make it a function:

export const RecordRoute = id => '/record/' + id;

then you use it like

`history.push(RecordRoute(item.id));`